test/cpp/end2end/xds/xds_end2end_test_lib.h

```diff
@@ -747,6 +748,11 @@ class XdsEnd2endTest : public ::testing::TestWithParam<XdsTestType> {
       return *this;
     }
 
+    RpcOptions& set_timeout(grpc_core::Duration timeout) {
```

Comment:
Why not replace the `set_timeout_ms(int rpc_timeout_ms)` version with this? Are both needed?

Reply:
Yeah, they're both useful.  The vast majority of the tests just hard-code a millisecond value as an integer, and it makes the test less verbose to use this form.